indexer/osm_editor.hpp

```diff
@@ -165,6 +165,7 @@ class Editor final : public MwmSet::Observer
   // Use GetFeatureStatus(fid) instead. This function is used when a feature is
   // not yet saved and we have to know if it was modified or created.
   static bool IsCreatedFeature(FeatureID const & fid);
+  bool WasDefaultNameSaved(FeatureID const & fid) const;
```

Comment:
This method returns true in case when original feature from mwm has default name. Comment added